DiepSam_html/index.html

```diff
@@ -0,0 +1,286 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="name" content="width=device-width, initial-scale=1">
+  <title>Design DiepSam</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <header class="header">
+    <div class="container">
+      <a class="main-logo" href="#"><img src="images/logo.png"></a>
+      <nav class="navigation">
+        <ul class="main-menu">
+          <li class="menu-item"><a href="#">HOME</a></li>
+          <li class="menu-item"><a href="#">ABOUT US</a></li>
+          <li class="menu-item"><a href="#">PRODUCT</a></li>
+          <li class="menu-item"><a href="#">PROJECTS</a></li>
+          <li class="menu-item"><a href="#">NEW-EVENT</a></li>
+          <li class="menu-item"><a href="#">GALLERY</a></li>
+          <li class="menu-item"><a href="#">CONTACT</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <main class="main-content">
+    <section class="banner">
+      <div class="container">
+        <h3>quality policy</h3>
+        <h1>commitment to product quality</h1>
+        <div class="btn-primary">
+          <a href="#">view more</a>
+        </div>
+      </div>
+    </section>
+    <section class="about-us">
+      <div class="container">
+        <div class="section-head">
+          <h3 class="section-title">about us</h3>
+          <div class="logo">
```

Comment:
- Using name have the prefix in this case.
- Ex: `section-title-icon`
 - Apply for all section